server/src/jobs/offrePartenaire/validateComputedJobPartners.ts

```diff
@@ -52,7 +55,11 @@ export const validateComputedJobPartners = async () => {
               },
             })
           } else {
-            counters.success++
+            if (validated) {
+              counters.success++
+            } else {
+              counters.error++
+            }
```

Comment:
```suggestion
       counters.success++
```
Le typage garantit que `validated = true` tout le temps ici

Reply:
en effet, c'est un reliquat de ma première version que j'ai oublié d'enlever